deploy/charts/openebs-monitoring/dashboards/cStor/openEBS-main-dashboard.json

```diff
@@ -0,0 +1,2187 @@
+{
```

Comment:
Convert the dashboard json file `openEBS-main-dashboard -> openebs-main-dashboard` name to lowercase or else installation will fail.

Reply:
Done